river/datasets/synth/friedman.py

```diff
@@ -51,6 +52,230 @@ def __iter__(self):
         while True:
 
             x = {i: rng.uniform(a=0, b=1) for i in range(10)}
-            y = 10 * math.sin(math.pi * x[0] * x[1]) + 20 * (x[2] - .5) ** 2 + 10 * x[3] + 5 * x[4]
+            y = 10 * math.sin(math.pi * x[0] * x[1]) + 20 * (x[2] - .5) ** 2 + 10 * x[3] \
+                + 5 * x[4] + rng.gauss(mu=0, sigma=1)
 
             yield x, y
+
+
+class FriedmanDrift(base.SyntheticDataset):
```

Comment:
Should this class extend the `Friedman` class?

Reply:
Well, apart from clearly stating that `FriedmanDrift` is an extension of `Friedman`, there are no immediate gains. However, I think this is enough reason by itself.

Changed in commit commit [afa307d](https://github.com/online-ml/river/pull/387/commits/afa307da973ab1519b2d06ae3852ba566f5ee147).